docs/proposals/20201125-jaeger-traces-store.md

```diff
@@ -0,0 +1,122 @@
+
+#### Single Jaeger instance
+
+This deployment topology deploys a single Jaeger instance for all tenants with a single Elasticsearch instance. The data for each tenant would contain a unique attribute to identify a tenant. The label would be dynamically injected in the Observatorium API service or in the OpenTelemetry collector.
+
+This deployment strategy will not work with the current Jaeger components. Here are few issues, and the list might not be complete:
+* Service and operation name API. The service query API nor storage does not expose/store labels.
+* Dependency/service architecture diagram. To support this feature dependency schema, query, collector and Spark aggregation would have to change.
+* Data retention/TTL configurable pre tenant. The data retention in Elasticsearch is configurable per index.
+
+##### OpenTelemetry collector configuration
+
+The OpenTelemetry collector is configured with a single exporter to export data to a single Jaeger collector.
+
+#### Jaeger instance per tenant
+
+This deployment topology deploys a Jaeger instance per tenant and all Jaeger instances use a single Elasticsearch instance. The data for each tenant is stored in a separate Elasticsearch index. Jaeger was designed to separate tenant data per index. Therefore, all Jaeger functionality works well with this deployment strategy.
+
+##### OpenTelemetry collector configuration
+
+The OpenTelemetry collector is configured with an exporter per tenant that exports data to Jaeger collector allocated per a single tenant.
+
+### Jaeger Query
+
+There are two ways users could access data stored in Jaeger:
+1. Expose only Jaeger Query API in the Observatorium API.
+2. Expose Jaeger UI outside the cluster.
+
+#### Expose only Jaeger Query API
+
+Jaeger query API would be exposed in the Observatorium API and users would have to deploy Jaeger UI and use Observatorium API as a data source.
+
+Jaeger query exposes multiple HTTP and gRPC APIs. The Jaeger UI uses HTTP API. This API does not have OpenAPI schema, and it is only defined in [http_handler.go](https://github.com/jaegertracing/jaeger/blob/0dd3e2da0579caed9e24ad2782a1f638ad63214d/cmd/query/app/http_handler.go#L119). The API uses this [data model](https://github.com/jaegertracing/jaeger/blob/9cd7a7ec1aa43b24a8a970eb5b393ca2ffd98a5d/model/json/model.go#L52).
+
+#### Expose Jaeger UI
+
+In addition to exposing Jaeger query API the Observatorium could deploy Jaeger UI as well. At the moment Observatorium does not expose any UI and it is expected that users deploy UI locally.
+
```

Comment:
Would be awesome to elaborate on alternatives. I see one major alternative: Tempo. Even you are against of it since for first glance it sounds like a new solution and lacking indexing, do you mind starting some alternative section about it and we can expand it together in terms of pros / cons